src/cmd/services/m3query/config/config.go

```diff
@@ -155,6 +156,22 @@ type FilterConfiguration struct {
 	CompleteTags Filter `yaml:"completeTags"`
 }
 
+// ResultOptions are the result options for query.
+type ResultOptions struct {
+	// DropNaNsInResults drops NaNs before returning query results.
+	// The default is true, which matches Prometheus
+	DropNaNsInResults *bool `yaml:"dropNaNs"`
+}
+
+// DropNaNsOrDefault returns the provided input or default if none is provided
+func (c ResultOptions) DropNaNsOrDefault() *bool {
```

Comment:
This should probably return a `bool` not a `*bool`

Reply:
Shoot yeah meant to change that.